### PR TITLE
feat: get ticket button added

### LINKED ISSUE #185: add get tickets in venue page
get tickets option is not showing in the venue page it is only showing on the home page. add it to venue also

Screenshot 2023-09-04 at 9 24 04 PM

### PR DESCRIPTION
**Pull request by vishvamsinh28**
added button on venue page to buy ticket 

Screenshot 2023-09-04 at 9 54 00 PM

fixes #185

### CODE REVIEW

**Comment by AceTheCreator** [comment]
@vishvamsinh28 I don't see an issue opened for this pr; we're also following the design provided by @Mayaleeeee 

@Mayaleeeee wyt?

**Comment by vishvamsinh28** [comment]
@AceTheCreator  I have opened issue for this #186 . And @derberg wanted button for buying tickets on the venue page also

**Comment by Mayaleeeee** [comment]
> @vishvamsinh28 I don't see an issue opened for this pr; we're also following the design provided by @Mayaleeeee
> 
> @Mayaleeeee wyt?

It makes sense, @AceTheCreator. The button has to be there because we want to make it easy for our users to buy tickets for each location without returning to the main landing page. So, a big thank you to @derberg and @vishvamsinh28 for their contribution.

@vishvamsinh28, let me explain how it's supposed to work. We need to remove the ''**Agenda, Speakers, and Event & Host Sponsor''** button. 

Please watch the video below for a visual reference on how it should look and how the button should interact.
 [screen-recording (2).webm](https://github.com/asyncapi/conference-website/assets/105395613/558fd397-19ef-42d0-96a0-cf836ada8382) and here's the Figma [file](https://www.figma.com/file/sS5ofLRtuFqDGFvlNBurns/AsyncAPI-Conference-on-Tour'23%2FAACoT?type=design&node-id=1360%3A1063&mode=design&t=e1GFZkbSFOLU6Dhh-1).

**Comment by vishvamsinh28** [comment]
@Mayaleeeee @AceTheCreator removed buttons and updated design of ticket button

**Comment by AceTheCreator** [comment]
@Mayaleeeee WYT?

**Comment by thulieblack** [comment]
@vishvamsinh28, please could you add another **Free Tickets Form** button on the London page.
 
 here's the link 👉🏿 https://forms.gle/c7t8k2JABcF26guq5
 
 thank you so much

**Comment by Mayaleeeee** [comment]
> @Mayaleeeee WYT?

Let's use our pry colour as suggested by @thulieblack.
Thanks for your contribution, @vishvamsinh28 
Screenshot 2023-09-06 214735

**Comment by Mayaleeeee** [comment]
@AceTheCreator Is the distance between the location and date intentional? I would like to know before creating issues on it.

Screenshot 2023-09-06 215040

**Comment by thulieblack** [comment]
@vishvamsinh28, you made a mistake on the London Page.

There need to be two buttons:

 1. **Get your Tickets** ➡️ https://opencollective.com/asyncapi/events/asyncapi-conference-on-tour-6b3c0aa1/contribute/aacot-london-edition-61313
 2. **Free Tickets Form** ➡️ https://docs.google.com/forms/d/e/1FAIpQLSdKxKvHGqmCmFJ7ldJJcAJNnmuie057vYrEh219gCXRDMZU5g/viewform

**Comment by vishvamsinh28** [comment]
@thulieblack oh sorry lemme fix it

**Comment by vishvamsinh28** [comment]
@thulieblack buttons and links are updated

File: config/city-lists.json [hunk 1]
```diff
@@ -6,7 +6,13 @@
...
 		"img": "/img/london.jpeg",
 		"address": "IBM UK Limited 20 York Road London SE1 7ND",
-		"sponsors": ["/img/IBM.png"]
+		"sponsors": [
+			"/img/IBM.png"
+		],
+		"ticket": "https://opencollective.com/asyncapi/events/asyncapi-conference-on-tour-6b3c0aa1/contribute/aacot-london-edition-61313",
+		"freeTicket":"https://docs.google.com/forms/d/e/1FAIpQLSdKxKvHGqmCmFJ7ldJJcAJNnmuie057vYrEh219gCXRDMZU5g/viewform",
+		"buttonText":"Get your Tickets",
+		"freeTicketTxt":"Tickets Form"
```
Comment (thulieblack) [inline config/city-lists.json:15]:
```suggestion
		"freeTicketTxt":"Free Tickets Form"
```

**Comment by Mayaleeeee** [comment]
Hey @thulieblack 
We can't have the two tickets like this


And what's the difference between "Get your Tickets" and the "Ticket Form"

**Comment by thulieblack** [comment]
@Mayaleeeee, it is correct the way it is. We have two different buttons for 2 different tickets

We have a paid ticket option and a free ticket option that's why the form is there

**Comment by Mayaleeeee** [comment]
> @Mayaleeeee, it is correct the way it is. We have two different buttons for 2 different tickets
> 
> We have a paid ticket option and a free ticket option that's why the form is there

We can't have two primary buttons sitting very close to each other on the interface because it's confusing and distracting.

Primary buttons are used to represent the most important action on a page, so what I am saying in essence is that, one has to be a pry button, and the other a secondary.

**Comment by thulieblack** [comment]
@Mayaleeeee, do you have suggestions on what we should do because they both need to be there on the page?

**Comment by Mayaleeeee** [comment]
> @Mayaleeeee, do you have suggestions on what we should do because they both need to be there on the page?

Yeah, I do.

Which one is our main priority?
**Get Your Tickets** or **Ticket Form**?

If it's the first one, then that would be our primary button, and the second would be a secondary.

**Comment by thulieblack** [comment]
Prioritize **Get your Tickets** the following **Free Tickets Form**

**Comment by vishvamsinh28** [comment]
@Mayaleeeee How it should look on the website ?

**Comment by Mayaleeeee** [comment]
> @Mayaleeeee How it should look on the website ?

I'm gonna design that, and show you.

Thank you @thulieblack @vishvamsinh28

**Comment by Mayaleeeee** [comment]
> @Mayaleeeee, do you have suggestions on what we should do because they both need to be there on the page?

@thulieblack, So here are my suggestions.

1. The button below is the secondary button design. However, it may not be evident to users. I want to be sure it's not just from my side, so please let me know your thoughts on its visibility.


2. We can make the "Ticket Form'' a white fill; that would be perfect for visibility. 
Also, I suggest we change the name of the **"Tickets Form"** to **"Grab Free Tickets"** to clarify that users can obtain a free ticket by filling out the form. The current name gives the impression that users need to acquire a ticket (from the **Get your Tickets button**) or complete a form to get a paid or free ticket (from the **Ticket Form button**).  
By using clear and concise language, we can provide better instructions to users and ensure they understand what is expected of them. 

**In essence, I am saying that if the **"Ticket Form"** button is for free tickets, then let it be explicitly mentioned to avoid confusion and misunderstandings.**

**Comment by thulieblack** [comment]
Hey Maya, I like the design; let's go ahead with it.

@vishvamsinh28 These changes only apply to the London page only. The Madrid page is fine as it is. You can also change the button description to **"Grab Free Tickets"** as Maya suggests. 

The form will be visible on the website until the 17th of September so it is only for a week that it needs to be there.

Thanks

**Comment by thulieblack** [comment]
Hey @vishvamsinh28 do not add the free ticket button anymore . Just remove it whenever you get a chance and lets get it merged

**Comment by vishvamsinh28** [comment]
@thulieblack we can merge this pr now i have made changes

**Comment by thulieblack** [comment]
hey @vishvamsinh28 we're almost there a few things
1. Update your branch with the latest commits
2. Return the Free Tickets Form to the Madrid page. We are only making changes to the London Page.

Let me know if I can clear up any confusion or need any help

**Comment by thulieblack** [comment]
Thanks so much @AceTheCreator please merge this

**Pull request merged**